Directory.Build.props

```diff
@@ -17,6 +16,7 @@
     <WarningsAsErrors>nullable</WarningsAsErrors>
     <ImplicitUsings>enable</ImplicitUsings>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
```

Comment:
This removes the following warning: `This project cannot be packaged because packaging has been disabled. Add <IsPackable>true</IsPackable> to the project file to enable producing a package from this project.`, which is correct, as we don't want some projects (like `Umbraco.Web.UI`) to generate a NuGet package.